pkg/commands/process/settings/rules.go

```diff
@@ -240,3 +275,21 @@ func buildRules(definitions map[string]RuleDefinition, enabledRules map[string]s
 
 	return rules
 }
+
+func cachedRulesExist(bearerRulesDir string) bool {
+	_, err := os.Stat(bearerRulesDir)
+	return err == nil
+}
+
+func cleanupRuleDirFiles(bearerRulesDir string) error {
+	os.RemoveAll(bearerRulesDir)
+	return nil
+}
+
+func bearerRulesDir() string {
+	tmpDir := os.TempDir()
+	if tmpDir == "/tmp" {
+		return "/tmp/bearer-rules/"
+	}
+	return tmpDir + "bearer-rules"
```

Comment:
Do we need the path separator here?
```suggestion
	return tmpDir + "/bearer-rules"
```
(or filepath.Join)

Reply:
Yeah it's a bit weird. From what I can tell with `os.TempDir()` is that if a temp directory exists, the trailing slash is included (at least for my OS), but if there's no temp directory, it returns `/tmp` without a trailing slash.. 

Maybe best to do a check and add it if it's missing? What do you think?

Reply:
It looks like `filepath.Join` will handle the differences:
```go
	fmt.Println(filepath.Join("/tmp", "rules")) //      /tmp/rules
	fmt.Println(filepath.Join("/tmp/", "rules")) //     /tmp/rules
```

Reply:
Ah, brilliant, thanks! 🤓